index.js

```diff
@@ -97,7 +97,36 @@ export default class Delaunator {
         let i2y = coords[2 * i2 + 1];
 
         if (minRadius === Infinity) {
-            throw new Error('No Delaunay triangulation exists for this input.');
+            // order collinear points
+            // and return the list as a hull
+            let a = 0, b = 0;
+            for (let i = 1; i < n; i++) {
+                if ((a = coords[2 * i] - coords[0]) || (b = coords[2 * i + 1] - coords[1])) {
+                    const norm = Math.sqrt(a * a + b * b);
+                    a /= norm;
+                    b /= norm;
```

Comment:
Maybe let's rename to `dx`/`dy` to make it easier to understand what happens here?

Reply:
Indeed, the normalization was useless; I think it's clearer now.